examples/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java

```diff
@@ -218,8 +218,8 @@ public void onCompleted() {
 
     try {
       RouteNote[] requests =
-          {newNote("First message", 0, 0), newNote("Second message", 0, 1),
-              newNote("Third message", 1, 0), newNote("Fourth message", 1, 1)};
+          {newNote("First message", 0, 0), newNote("Second message", 0, 10000000),
+              newNote("Third message", 10000000, 0), newNote("Fourth message", 10000000, 10000000)};
```

Comment:
nit: it's hard to count how many zeros are there. `10_000_000` might be better.

Reply:
Done